src/renderers/native/ReactNative/ReactNativeContainerInfo.js

```diff
@@ -11,7 +11,7 @@
  */
 'use strict';
 
-function ReactNativeContainerInfo(tag) {
+function ReactNativeContainerInfo(tag : number) {
```

Comment:
> Ideally we should lint for Flow style guides (like no space before colon).

I'll be the linter right now and tell you that you have a space before the colon.